src/fn_call.rs

```diff
@@ -350,7 +356,10 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a+'mir>: crate::MiriEvalContextExt<'a,
                     let name_ptr = this.read_scalar(args[0])?.not_undef()?;
                     if !name_ptr.is_null_ptr(this) {
                         let name_ptr = name_ptr.to_ptr()?;
-                        let name = this.memory().get(name_ptr.alloc_id)?.read_c_str(tcx, name_ptr)?.to_owned();
+                        let name = this.memory()
+                                       .get(name_ptr.alloc_id)?
```

Comment:
[Style guide says to use block indent](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/expressions.md#chains-of-fields-and-method-calls)